provider/aws/aws.go

```diff
@@ -36,6 +36,8 @@ import (
 	"sigs.k8s.io/external-dns/endpoint"
 	"sigs.k8s.io/external-dns/plan"
 	"sigs.k8s.io/external-dns/provider"
+
+	"go.uber.org/ratelimit"
```

Comment:
Why is this in a new import group?

Reply:
I saw that it had it's own import group in the OVH provider and decided to follow its example. Do you suggest putting it in the second (github.com) import group instead?